### ISSUE TITLE
loadDashboard resets filters

### ISSUE THREAD

**Issue by kajyr** [looker-open-source/embed-sdk#96]
Hi! I noticed that `.loadDashboard` resets the filters applied with the `.updateFilters` call, it is intended?

Let me explain better:

I have 2 dashboards, set up with the same filters. I embed the first

```
LookerEmbedSDK.createDashboardWithId(dashboard)
    .appendTo("#dashboard")
    .withNext()
    .withFilters({...})
    .build()
    .connect()
    .then((dashboard: LookerEmbedDashboard) => {
      return dashboard;
    });
```

I then change some toggles in the page, and call `dashboard.updateFilters({...})`.

Now if I change the dashboard to the second "page" with `dashboard.loadDashboard(otherDashboard)`
 I see that the new dashboard does not have the new filter set, but it went to the default setup in Looker.

In case this is not a bug, what is the suggested way to bring the same filters across two (compatible) dashboards?

Thanks!

**Comment by EricNelson12** [comment]
This may or may not be related, but calling `LookerEmbedSDK.createDashboardWithUrl(mySignedUrlWithSomeParams)` and subsequently calling `dashboard.loadDashboard(otherDashboard)` will cause params like `hide_filters` to not be applied to the newly loaded dashboard.

Maybe `loadDashboard` should support `withParams` ?

**Comment by jariahh** [comment]
I agree, this feels like a major miss. I have to let the new dashboard load, then set the filters and any other options, which causes a long delay for users to see the data they want! Additionally, it makes additional calls to the queries in BigQuery causing more requests and higher costs!
